src/devices/x10_sec.c

```diff
@@ -0,0 +1,137 @@
+/*
+ * X10 Security sensor decoding for rtl_433
+ *
+ * Based on code provided by Willi 'wherzig' in issue #30 (2014-04-21)
+ * https://github.com/merbanan/rtl_433/issues/30
+ * Danke, Willi
+ *
+ * Tested with American sensors operating at 310 MHz
+ * e.g., rtl_433 -f 310.558M
+ *
+ * This is pretty rudimentary, and I bet the byte value decoding, based
+ * on limited observations, doesn't take into account bits that might
+ * be set to indicate something like a low battery condition.
+ *
+ * Copyright (C) 2018 Anthony Kava
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ */
+
+#include "rtl_433.h"
+#include "data.h"
+#include "util.h"
+
+static int x10_sec_callback(bitbuffer_t *bitbuffer)
+{
+	bitrow_t *bb = bitbuffer->bb;		
+	data_t *data;
+
+	char *remarks_str="UNKNOWN";		/* human-readable remarks */
+	char time_str[LOCAL_TIME_BUFLEN];
+	char x10_id_str[5]="\0";			/* string showing hex value */
+	char x10_code_str[5]="\0";			/* string showing hex value */
+
+	/* validate what we received ...
+	 * used intermediate variables test_a and test_b to suppress
+	 * warnings generated when compiling using:
+	 *
+	 *   if((bb[1][0]^0x0f)==bb[1][1] && (bb[1][2]^0xff)==bb[1][3])
+	 */
+	uint8_t test_a=bb[1][0]^0x0f;
+	uint8_t test_b=bb[1][2]^0xff;
+	if(test_a==bb[1][1] && test_b==bb[1][3])
+	{
+		/* set remarks based on code received */
+        switch(bb[1][2])
```

Comment:
add a space between keywords and parens `if (…`

Reply:
Adding spaces